gstd-meta/src/prelude.rs

```diff
@@ -0,0 +1,24 @@
+// This file is part of Gear.
```

Comment:
Do we really need a dedicated prelude for `gstd-meta`?

Reply:
we don't

Reply:
why is it marked as resolved when module is still here?

Reply:
Removed